site/docs/config.toml

```diff
@@ -52,6 +52,7 @@ ignoreFiles = [
   "/opentitan-docs/",
   "/scratch/",
   "/site/*",
+  "/src/",
```

Comment:
No need for that either without the modified chipwhisperer package installation.

Reply:
Done.